src/assets/a11y-viewer.css

```diff
@@ -239,14 +269,15 @@ iframe {
 /* navigator */
 @media screen and (min-width: 1055px) {
     .app-header .url-navigator {
+        width: calc(25% - 40px);
         float: right;
-        margin-right: 1rem;
     }
 }
 
 .url-navigator input {
     padding: 0 .5rem;
     height: 2.5rem;
+    width: calc(100% - 64px);
```

Comment:
is 64px the button width? The button width is not set, it is 64px because of the content inside. This can change and thus will the 64px break